app/redux/wallet/actions.js

```diff
@@ -213,7 +192,13 @@ export const setCurrentAccount = ({ index }: { index: number }): Action => ({ ty
 
 export const setMnemonic = ({ mnemonic }: { mnemonic: string }): Action => ({ type: SET_MNEMONIC, payload: { mnemonic } });
 
-export const setTransactions = ({ transactions }: { transactions: TxList }): Action => ({ type: SET_TRANSACTIONS, payload: { transactions } });
+export const setTransactions = ({ transactions }: { transactions: TxList }): Action => {
+  const sortedTransactions = {};
+  Object.keys(transactions).forEach((walletKey: string) => {
```

Comment:
no sorting should be done here, it's pure state update function with no side effects on data

Reply:
also I failed to find any reason for this sorting

Reply:
removed